### PR TITLE
Fixes in documentation for wordembedding

### PR DESCRIPTION
**Pull request by Ivanidzo4ka**
Better example, and replace html encoding to xml encoding

### CODE REVIEW

File: src/Microsoft.ML.Transforms/Text/doc.xml [hunk 1]
```diff
@@ -192,10 +192,10 @@
         WordEmbeddings wrap different embedding models, such as GloVe. Users can specify which embedding to use. 
         The available options are various versions of <a href="https://nlp.stanford.edu/projects/glove/">GloVe Models</a>, <a href="https://en.wikipedia.org/wiki/FastText">fastText</a>, and <a href="http://anthology.aclweb.org/P/P14/P14-1146.pdf">SSWE</a>.
         <para>
-          Note: As WordEmbedding requires a column with text vector, e.g. %3C%27this%27, %27is%27, %27good%27%3E, users need to create an input column by
-          using the output_tokens=True for TextTransform to convert a column with sentences like "This is good" into %3C%27this%27, %27is%27, %27good%27 %3E.
-          The suffix of %27_TransformedText%27 is added to the original column name to create the output token column. For instance if the input column is %27body%27,
-          the output tokens column is named %27body_TransformedText%27.
```
Comment (sfilipi) [inline src/Microsoft.ML.Transforms/Text/doc.xml]: I have fixed those in #625, but you are also changing the example.

File: src/Microsoft.ML.Transforms/Text/doc.xml [hunk 2]
```diff
@@ -204,7 +204,12 @@
               <description>
                 &quot;fastText Wikipedia 300D&quot; by Facebook, Inc. is licensed under <a href="https://creativecommons.org/licenses/by-sa/3.0/">CC-BY-SA 3.0</a> based on:
                 P. Bojanowski*, E. Grave*, A. Joulin, T. Mikolov,<a href="https://arxiv.org/abs/1607.04606">Enriching Word Vectors with Subword Information</a>
-                %40article%7Bbojanowski2016enriching%2C%0A%20%20title%3D%7BEnriching%20Word%20Vectors%20with%20Subword%20Information%7D%2C%0A%20%20author%3D%7BBojanowski%2C%20Piotr%20and%20Grave%2C%20Edouard%20and%20Joulin%2C%20Armand%20and%20Mikolov%2C%20Tomas%7D%2C%0A%20%20journal%3D%7BarXiv%20preprint%20arXiv%3A1607.04606%7D%2C%0A%20%20year%3D%7B2016%7D%0A%7D
+                @article{bojanowski2016enriching,
+  title={Enriching Word Vectors with Subword Information},
+  author={Bojanowski, Piotr and Grave, Edouard and Joulin, Armand and Mikolov, Tomas},
+  journal={arXiv preprint arXiv:1607.04606},
+  year={2016}
```
Comment (sfilipi) [inline src/Microsoft.ML.Transforms/Text/doc.xml]: not sure how well will this appear.

File: src/Microsoft.ML.Transforms/Text/doc.xml [hunk 3]
```diff
@@ -192,10 +192,10 @@
...
         The available options are various versions of <a href="https://nlp.stanford.edu/projects/glove/">GloVe Models</a>, <a href="https://en.wikipedia.org/wiki/FastText">fastText</a>, and <a href="http://anthology.aclweb.org/P/P14/P14-1146.pdf">SSWE</a>.
         <para>
-          Note: As WordEmbedding requires a column with text vector, e.g. %3C%27this%27, %27is%27, %27good%27%3E, users need to create an input column by
-          using the output_tokens=True for TextTransform to convert a column with sentences like "This is good" into %3C%27this%27, %27is%27, %27good%27 %3E.
-          The suffix of %27_TransformedText%27 is added to the original column name to create the output token column. For instance if the input column is %27body%27,
-          the output tokens column is named %27body_TransformedText%27.
+          Note: As WordEmbedding requires a column with text vector, e.g. &lt;&apos;this&apos;, &apos;is&apos;, &apos;good&apos;&gt;, users need to create an input column by
+          using the output_tokens=True for TextTransform to convert a column with sentences like "This is good" into &lt;&apos;this&apos;, &apos;is&apos;, &apos;good&apos; &gt;.
```
Comment (zeahmed) [inline src/Microsoft.ML.Transforms/Text/doc.xml]: What if some one does not want to use TextTransform for creating vector of text. I would also recommend mentioning of wordtokenizer transform here.

Comment (zeahmed) [reply src/Microsoft.ML.Transforms/Text/doc.xml]: Not sure if its related to your change...:)

File: src/Microsoft.ML.Transforms/Text/doc.xml (hunk 1 above)
Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/Text/doc.xml]: Feel free to take it :)

File: src/Microsoft.ML.Transforms/Text/doc.xml [hunk 4]
```diff
@@ -221,7 +226,14 @@
     <example name="WordEmbeddings">
       <example>
         <code language="csharp">
-          pipeline.Add(new WordEmbeddings((&quot;InVectorTextCol&quot; , &quot;OutTextCol&quot;)));
+            pipeline.Add(new TextFeaturizer(&quot;Words&quot;, &quot;InputTextCol&quot;)
+            {
+                TextCase = TextNormalizerTransformCaseNormalizationMode.Lower,
+                OutputTokens = true,
```
Comment (zeahmed) [inline src/Microsoft.ML.Transforms/Text/doc.xml]: I think it looks bit odd here to use TextFeaturizer for tokenization only. Wouldn't it be nice to use simply wordtokenizer in this example.

Comment (Ivanidzo4ka) [reply src/Microsoft.ML.Transforms/Text/doc.xml]: Yes, it's weird, but Wordtokenizer doesn't do lower casing for words and AFAIK all models use lowercased words.

Comment (zeahmed) [reply src/Microsoft.ML.Transforms/Text/doc.xml]: We have TextNormalizer for that but I don't find it in CSharpApi.cs. Is it exposed in public API?

Comment (justinormont) [reply src/Microsoft.ML.Transforms/Text/doc.xml]: Using the TextFeaturizer is the correct thing to do. As @Ivanidzo4ka says, we need the lower casing, punctuation removal, and (sometimes) the stop word removal.

And in most cases you want to get the ngrams at the same time, as this is the winning method (wordembeddings + ngrams + chargrams).

**Pull request closed without merging**